src/specs/seqn-printer.lisp

```diff
@@ -0,0 +1,32 @@
+(in-package #:geb.seqn.spec)
+
+(defmacro easy-printer (class-name)
+  `(defmethod print-object ((obj ,class-name) stream)
+     (format stream "~A"
+             (cons ',class-name
+                   (mapcar #'cdr (geb.mixins:to-pointwise-list obj))))))
```

Comment:
If we are copy and pasting this, maybe we should make a separate topic that makes this in the utils or something.

Reply:
Good idea